lib/tessel/deployment/javascript.js

```diff
@@ -379,7 +379,10 @@ exportables.injectBinaryModules = function(globRoot, tempBundlePath, options) {
 };
 
 exportables.preBundle = function(options) {
-  return exportables.resolveBinaryModules(options);
+  return options.tessel.fetchCurrentNodeVersion().then(version => {
+    options.version = version;
```

Comment:
This should be `nodeVersion` to match its naming in the code for the `t2 version` command